go/roothash/api/api.go

```diff
@@ -193,3 +194,74 @@ type TransactionSchedulerParameters struct {
 	// in bytes.
 	MaxBatchSizeBytes uint64 `json:"max_batch_size_bytes"`
 }
+
+// SanityCheck does basic sanity checking on the genesis state.
+func (g *Genesis) SanityCheck() error {
+	if g.Parameters.RoundTimeout < 1*time.Second {
+		return fmt.Errorf("roothash: sanity check failed: round timeout must be >= 1 second")
+	}
+
+	if g.Parameters.TransactionScheduler.Algorithm != "batching" {
```

Comment:
constify "batching"

Reply:
I couldn't do this nicely because of import cycles, but I added a comment to point to the actual constant that should be used here.

Reply:
Maybe define the valid algorithm names here (e.g., in `go/roothash/api`) instead and use the constant in the worker which needs to import it anyway?